reference-architecture/vmware-ansible/roles/openshift-registry/tasks/main.yaml

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
You should be able to use the default registry/router roles. @abutcher can provide help with setting the vars if needed.